code/src/main/java/frc/robot/RobotConfig.java

```diff
@@ -38,14 +49,24 @@
         public final int frontRightAbsoluteEncoder = 1;
         public final int rearLeftAbsoluteEncoder = 3;
         public final int rearRightAbsoluteEncoder = 2;
-        public final double swerveX = .67 / 2;
-        public final double swerveY = .25;
     }
 
     public final class Physical {
-        public final double widthInCms = 0;
-        public final double lengthInCms = 0;
-        public final double weightInKgs = 0;
+        /*
+         * When using these values for the swerve drive, make sure to divide them by two
```

Comment:
these values are relative to the center of the bot. we already divided them by 2

Reply:
look again.  we have the length and width of the drivetrain in meteres.  we divide them by two and change the sign of the values when we use them in the SwerveDrivetrain